response_operations_ui/templates/conversation-view/conversation-view.html

```diff
@@ -16,7 +16,7 @@
     {% include 'conversation-view/cv-handle-errors.html' %}
     {% include 'conversation-view/cv-back-link.html' %}
 
-    <h1 class="saturn u-mt-s" name="page-messages-title">{{ subject }}</h1>
+    <h1 class="u-fs-l u-mt-s" name="page-messages-title">{{ subject }}</h1>
```

Comment:
The difference in this from master is that the subject box isn't the same length as the message box

Reply:
Fixed this by adding `text--block` to the input field below

Reply:
👍